pages/studentPortal/labs/multiplication/game.tsx

```diff
@@ -1,14 +1,114 @@
-import MultiplicationBlock from "../../../../components/math/MultiplicationBlock";
-//create big box
+import React, { useState } from "react";
+import MultiplicationBlock, {
+  BlockState,
+} from "../../../../components/math/MultiplicationBlock";
+import { Button } from "../../../../components/ui/Button";
+
 export default function BlockComponentGallery() {
+  const [player, setPlayer] = useState(false);
```

Comment:
I changed it to isPlayerOneActive.

How would you do the second if you need to increment up and down? I'd love to see that.  so it could go from 1 to 2 back to 1 on a button click.